### PR TITLE
Update distribution management for pom.xml

### PR DESCRIPTION
**Pull request by anwar6953**
Cherry picking https://github.com/caskdata/hydrator-plugins/pull/122 to release/1.2 branch.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -50,6 +50,21 @@
     </license>
   </licenses>
 
+  <distributionManagement>
+    <repository>
+      <id>sonatype.release</id>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
+    </repository>
+    <snapshotRepository>
```
Comment (albertshau) [inline pom.xml:58]: why do we need this?

Comment (anwar6953) [inline pom.xml:58]: Do you mean we do not need this on release branch? Or on develop either?

Comment (albertshau) [inline pom.xml:58]: oh i see this was applied to develop then cherry-picked

**Comment by albertshau** [comment]
:+1:

**Pull request merged**